utils/posts.js

```diff
@@ -0,0 +1,204 @@
+import fs from 'fs';
```

Comment:
Let's only add this file if/when it is needed.

Reply:
done